dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/task/AbstractJdbcTask.java

```diff
@@ -125,7 +128,7 @@ protected ResultSetExtractor<Void> newCsvResultSetExtractor(
           }
           printer.println();
         }
-        return null;
+        return new Summary(monitor.getCount(), interval);
```

Comment:
We introduce a `interval` argument to `newCsvResultSetExtractor` and we only use it to build the return value, maybe we could set the interval at one of the callers, see comment below on `JdbcSelectTask#doInConnection`